### PR TITLE
Azure Site Recovery - Updated example code

### PR DESCRIPTION
**Pull request by jawache**
#sign-off as per @TomArcher

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/site-recovery.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-9/docs-ref-services/site-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C17%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707171637536885-97%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@jawache  @johnpapa Merged

File: docs-ref-services/site-recovery.md [hunk 1]
```diff
@@ -13,26 +13,43 @@ ms.devlang: nodejs
...
 Learn more about [Azure Site Recovery](https://docs.microsoft.com/en-us/azure/site-recovery/site-recovery-overview).
 
-## Install the module with npm
-
-Use npm to install the Azure Site Recovery module for Node.js
-
-### Management
+## Management Package
 
+### Install npm module
```
Comment (johnpapa) [inline docs-ref-services/site-recovery.md]: module --> package

and add a sentence about this 

> Install the recovery services npm module

File: docs-ref-services/site-recovery.md [hunk 2]
```diff
@@ -13,26 +13,43 @@ ms.devlang: nodejs
...
 ```bash
 npm install azure-arm-recoveryservices
 ```
 
-##Example
+### Example
```
Comment (johnpapa) [inline docs-ref-services/site-recovery.md]: add a sentence to introduce what this example does

**Pull request merged**